tensorflow/compiler/xla/service/plaidml/tests/BUILD

```diff
@@ -74,11 +74,19 @@ tf_cc_test(
     ],
 )
 
+genrule(
+    name = "plaidml_conv_io_gen",
+    srcs = [],
+    outs = ["plaidml_conv_test_io.h"],
```

Comment:
Semantics: We generally use `.h.inc` as a suffix for genrules like this so that it's easier to distinguish. How do you feel about renaming this rule to `plaidml_conv_op_test_gen` and the output to `plaidml_conv_op_test.h.inc`?